### PR TITLE
add dev.Dockerfile to run in docker with local code/sdk/protocol

### PR DESCRIPTION
**Pull request by MarkTNO**

### CODE REVIEW

File: dev.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+RUN apt update && \
+    apt install -y libpq-dev python3-dev gcc make  # Install dependencies needed for psycopg2
+
+COPY orchestrator/requirements.txt /app/requirements.txt
+RUN pip install -r /app/requirements.txt --no-cache-dir
+
+COPY ../omotes-sdk-protocol/python/ /omotes-sdk-protocol/python/
+COPY ../omotes-sdk-python/ /omotes-sdk-python/
+RUN pip install -e /omotes-sdk-python/
```
Comment (cwang39403) [inline dev.Dockerfile:14]: Does this `-e` still have any effect here? As the packages have already been copied (and frozen) into the Docker filesystem. Will package changes still be reflected immediately in the running containers?

File: dev.Dockerfile
Comment (cwang39403) [inline dev.Dockerfile:1]: Maybe also add a short description in README when this `dev.Dockerfile` is used and for what purpose

**Review by cwang39403 — APPROVED** [review]
Approved with some optional comments and questions.

File: dev.Dockerfile (hunk 1 above)
Comment (lfse-slafleur) [reply dev.Dockerfile:14]: Good question. I suspect not. If a bind mount is used to overwrite /omotes-sdk-python/, then it could matter.

**Review by lfse-slafleur — APPROVED** [review]
Questions from Cheng-Kai still valid. Good addition!

File: dev.Dockerfile (hunk 1 above)
Comment (MarkTNO) [reply dev.Dockerfile]: I removed the `-e`

File: dev.Dockerfile
Comment (MarkTNO) [reply dev.Dockerfile:1]: see 5c1d426c908b91a33e9d8bdb6ce45cec0a42fec6

**Pull request merged**